internal/client/client.go

```diff
@@ -195,6 +195,16 @@ func (c *CSAPI) SendEventSynced(t *testing.T, roomID string, e b.Event) string {
 	return eventID
 }
 
+// SendToDevice sends an event of type `eventType` to `userID`'s device `deviceID` with `content`.
+func (c *CSAPI) SendToDevice(t *testing.T, eventType, userID, deviceID string, content map[string]interface{}) *http.Response {
+	return c.MustDoFunc(t, "PUT",
+		[]string{"_matrix", "client", "v3", "sendToDevice", eventType, strconv.Itoa(c.txnID)},
```

Comment:
Where is `c.txnID` incremented?

Reply:
FYI: I'm using the changes from `internal/client/client.go` in #412 and fixing this.